web/src/tippyjs.js

```diff
@@ -468,4 +468,24 @@ export function initialize() {
         delay: LONG_HOVER_DELAY,
         appendTo: () => document.body,
     });
+    delegate("body", {
+        target: ["#edit-button.disabled_setting_tooltip"],
+        content: $t({
+            defaultMessage: "This bot cannot be edited.",
+        }),
+        appendTo: () => document.body,
+        onHidden(instance) {
+            instance.destroy();
+        },
+    });
+    delegate("body", {
+        target: ["#deactivate-button.disabled_setting_tooltip"],
+        content: $t({
+            defaultMessage: "This bot cannot be deactivated.",
+        }),
+        appendTo: () => document.body,
+        onHidden(instance) {
+            instance.destroy();
+        },
```

Comment:
Instead of doing this, can we just use `data-tippy-content` attribute in the template? I don't think we would need to live-update these tooltips in any case, so we can just use `data-tippy-content` attribute maybe on the containing div.

Reply:
Yes it can be done. I have updated the PR.